jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -634,15 +643,38 @@ def serialize_script(script):
     else:
         return result
 
-
 def mk_multisig_script(pubs, k):
     """ Given a list of pubkeys and an integer k,
     construct a multisig script for k of N, where N is
     the length of the list `pubs`; script is returned
     as hex string.
     """
-    return serialize_script([k] + pubs + [len(pubs)]) + 'ae'
+    return serialize_script([k] + pubs + [len(pubs)]) \
+        + 'ae' #OP_CHECKMULTISIG
 
+def mk_freeze_script(pub, locktime):
+    """
+    Given a pubkey and locktime, create a script which can only be spent
+    after the locktime as passed using OP_CHECKLOCKTIMEVERIFY
```

Comment:
I think you mean "has passed" although amusingly it kind of parses as correct English, anyway :)